test/duct/logger/timbre_test.clj

```diff
@@ -78,7 +79,30 @@
       (logger/log logger :report ::testing)
       (is (re-matches
            #"(?x)
-             \d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\ 
-             REPORT\ \[duct\.logger\.timbre-test:78\]\ -\ 
+             \d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\
+             REPORT\ \[duct\.logger\.timbre-test:78\]\ -\
              :duct\.logger\.timbre-test/testing\n"
            (slurp tempfile))))))
+
+(deftest restore-root-timbre-config-test
+  (testing "manages taoensso.timbre/*config* binding"
+    (let [init-binding tao/*config*
+          config {::logger/timbre {:level :info, :appenders {}}}]
+
+      (assert (not= init-binding config)
```

Comment:
Sure, but if it's a problem then we can set it to something we know is different for the duration of the test.